app/controllers/registrar/dashboard_controller.rb

```diff
@@ -2,12 +2,6 @@ class Registrar
   class DashboardController < BaseController
     authorize_resource class: false
 
-    def show
-      if can?(:show, :poll)
-        redirect_to registrar_poll_url and return
```

Comment:
This changes the functionality a bit, are we sure this won't be considered a bug after next release? :)

Reply:
Yes, you're right. It alters it a bit. But the problem is that I needed to either test it properly, or abandon that code chunk. Though I wasn't sure that it was worth testing it, since it's not clear whether we need this functionality at all. All that `DashboardController`'s seem like useless. So I guess it's worth creating new ticket and solve that problem separately.

Reply:
Now it's fixed. `DashboardController` stuff is left for another ticket though.